tests/test_biophysical/test_animal/test_stocker/test_beef_stocker_config.py

```
@@ -0,0 +1,240 @@
+    """'pasture' and 'drylot_forage' must be accepted without raising."""
+    AnimalConfig._initialize_beef_stocker_config({"stocker_diet_system": valid_str})
+    assert AnimalConfig.stocker_diet_system.value == valid_str
+
+
+# ---------------------------------------------------------------------------
+# BeefPostWeaningDestination.STOCKER — no longer raises NotImplementedError
+# ---------------------------------------------------------------------------
+
+
+@pytest.mark.unit
+def test_stocker_post_weaning_destination_accepted(mocker: pytest_mock.MockerFixture) -> None:
```

Ruff (0.15.21)

[warning] Unused function argument: `mocker`

(ARG001)